packages/sdk/src/adapters/multipeer/rules.ts

```diff
@@ -1043,9 +989,6 @@ export const Rules: { [id in Payloads.PayloadType]: Rule } = {
                 client: Client,
                 message: Message<Payloads.UserJoined>
             ) => {
-                // Associate the client connection with the user id.
-                client.userId = message.payload.user.id;
```

Comment:
Why remove this?

Reply:
It is not removed, so much as moved. This field is now populated earlier than this, in the clientDesyncPreprocessor.

---
In reply to: [275434399](https://github.com/Microsoft/mixed-reality-extension-sdk/pull/303#discussion_r275434399) [](ancestors = 275434399)